### DISCUSSION TITLE
How to renew my expired domain on vercel?

### DISCUSSION THREAD

**Discussion by jeffrey125** [vercel/community#3243] [Help] [answered]
### Summary

# Renewing of Domain

Hi Everyone I'm having a hard time to look on the vercel platform on how to renew my vercel-bought domain. Also I have read the docs and according to them I should contact the support team for this. Thanks in advance.

![image](https://github.com/vercel/community/assets/65661841/03d93a77-8cd3-4348-b9ba-db65472c76ca)

### Steps to Reproduce

N/A

**Comment by amyegan** [comment, marked as answer]
Hey @jeffrey125. I can see that the renewal payment for [jeffreyasilo.com](http://jeffreyasilo.com/) has been failing for several days.

Fortunately, the domain is still within a grace period. Renewal will need to be processed manually.

Renewal for this domain, for 1 year, is priced at $20USD. If you are happy for us to proceed with this, please ensure that funds are available on the payment method configured within your Vercel UI. Then, reach out to our billing team by visiting https://vercel.com/help and selecting `Billing` from the drop-down on our contact form.

**Reply by jeffrey125** [reply]
Thanks for the answer, I will do this.

**Reply by amyegan** [reply]
I found your case and it looks like your domain is renewed now 🎉

**Reply by ismailelbadawy** [reply]
I have the same issue currently on a team called Icteam with the domain name csteam-elsafty.com I have already made a case a couple of days ago but haven't gotten any replies. Can you help me in any way ? @amyegan

**Reply by faraggi** [reply]
@ismailelbadawy @jeffrey125  I suppose you already resolved this. But in case anyone else sees this: 

The request has to be accepted by the receiver of the domain.

**Comment by lmf-git** [comment]
Contact/support does not let people register a case for anything like this, they have to put it under billing instead which is unfair to the billing team.
